src/doubly_linked.py

```diff
@@ -0,0 +1,70 @@
+from linked_list import LinkedList
+from linked_list import Node
+
+
+class DoublyLinked(LinkedList):
```

Comment:
Inheriting from LinkedList is not what we want to do here. We need to use composition. See http://learnpythonthehardway.org/book/ex44.html for one source of more information. The Doubly linked list is different enough from a linked list that it should be its own class. You're inheriting from LinkedList, but then just overwriting the methods.